homeassistant/components/google_assistant/const.py

```diff
@@ -124,6 +124,7 @@
 
 DEVICE_CLASS_TO_GOOGLE_TYPES = {
     (cover.DOMAIN, cover.DEVICE_CLASS_GARAGE): TYPE_GARAGE,
+    (cover.DOMAIN, cover.DEVICE_CLASS_GATE): TYPE_GARAGE,
```

Comment:
Oops. Missed this one when I fixed Google in https://github.com/home-assistant/core/pull/33541